### PR TITLE
Master windows app manifest

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

**Comment by mrward** [comment]
Can we make the FileTemplateTagsModifier internal instead of public?

**Comment by VincentDondain** [comment]
@mrward Fixed in 379bbd7f951e38f4f00656a33dc4fda76c29d2ac. Thanks.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/FileTemplateTagsModifier.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+using System;
+using MonoDevelop.Projects;
+using System.Collections.Generic;
+using System.IO;
+using MonoDevelop.Core;
+using System.Text;
+
+namespace MonoDevelop.Ide.Templates
+{
+	public static class FileTemplateTagsModifier
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/FileTemplateTagsModifier.cs]: Does this need to be public?

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/PropertyDescriptionTemplate.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+// THE SOFTWARE.
+using System;
+using System.Xml;
+using MonoDevelop.Core;
+using MonoDevelop.Ide.Templates;
+using MonoDevelop.Projects;
+
+namespace MonoDevelop.Ide.Templates
+{
+	public class PropertyDescriptionTemplate : FileDescriptionTemplate
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/PropertyDescriptionTemplate.cs]: Doesn't need to be public

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/FileTemplateTagsModifier.cs (hunk 1 above)
Comment (VincentDondain) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/FileTemplateTagsModifier.cs]: No that was a mistake and I fixed already 379bbd7f951e38f4f00656a33dc4fda76c29d2ac (:

**Pull request merged**